lib/resources/package.rb

```diff
@@ -288,7 +288,7 @@ def info(package_name)
       # Find the package
       cmd = inspec.command <<-EOF.gsub(/^\s*/, '')
         Get-ItemProperty (@("#{search_paths.join('", "')}") | Where-Object { Test-Path $_ }) |
-        Where-Object { $_.DisplayName -match "^\\s*#{package_name}\\s*$" -or $_.PSChildName -match "^\\s*#{package_name}\\s*$" } |
+	Where-Object { $_.DisplayName -match "^\\s*#{package_name.shellescape}\\s*$" -or $_.PSChildName -match "^\\s*#{package_name.shellescape}\\s*$" } |
```

Comment:
Slight nitpick, but if you could indent this line to match the others I would be most delighted 😄

Reply:
@jerryaldrichiii , argg!! seems a mistake from my side, fixed. :+1: